django_project/github_issue/static/js/github-issue.js

```diff
@@ -1,5 +1,5 @@
 //noinspection JSUnusedGlobalSymbols
-function submitIssue() {
+window.submitIssue = function () {
   $("#issue-submit-button").attr("disabled", "disabled");
   $.post("/github-issue", {
```

Comment:
This url won't work, you need to remove the slash, so it will be `$.post("github-issue")`.